modelmapper/src/test/java/com/baeldung/ModelMapperTest.java

```diff
@@ -0,0 +1,244 @@
+package com.baeldung;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertNotEquals;
+import static org.junit.jupiter.api.Assertions.assertNotNull;
+import static org.junit.jupiter.api.Assertions.assertNull;
+
+import com.bealdung.domain.Game;
+import com.bealdung.domain.GameMode;
+import com.bealdung.domain.GameSettings;
+import com.bealdung.domain.Player;
+import com.bealdung.dto.GameDTO;
+import com.bealdung.dto.PlayerDTO;
+import com.bealdung.repository.GameRepository;
+import java.time.Instant;
+import java.util.Collection;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.modelmapper.Condition;
+import org.modelmapper.Conditions;
+import org.modelmapper.Converter;
+import org.modelmapper.ModelMapper;
+import org.modelmapper.Provider;
+import org.modelmapper.TypeMap;
+import org.modelmapper.convention.MatchingStrategies;
+
+public class ModelMapperTest {
+
+  ModelMapper mapper;
+  GameRepository gameRepository;
+
+  @BeforeEach
+  public void setup() {
+    this.mapper = new ModelMapper();
+    this.gameRepository = new GameRepository();
+  }
+
+  @Test
+  public void whenMapGameWithExactMatch_convertsToDTO() {
```

Comment:
follow the naming convention whenX_thenY in the test names (you're missing "then")